azext_edge/edge/commands_dataflow.py

```diff
@@ -36,6 +36,44 @@ def list_dataflows(cmd, profile_name: str, instance_name: str, resource_group_na
     )
 
 
+def apply_dataflow(
+    cmd,
+    dataflow_name: str,
+    profile_name: str,
```

Comment:
Should profile_name default to "default" to save an input in simple cases? For broker related resources, defaulting to the "default" makes sense due to the nature of a singleton broker that gets created on AIO deployment. Dataflow profile may be less sticky but still could be convenience. Thoughts?

Reply:
make sense, will update!